gtsam/gtsam.i

```diff
@@ -2761,7 +2762,16 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtsam/slam/dataset.h>
+
 class SfmTrack {
+  SfmTrack();
+
+  double r;
```

Comment:
double r,g,b;

Reply:
The wrap parser doesn't support this syntax. :(